hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlClientTest.java

```diff
@@ -41,12 +43,52 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
 
 public class SqlClientTest extends SqlTestSupport {
```

Comment:
We should also try devise some test that checks that the correct coordinator was actually used. Perhaps we can add a metric for number of executed queries in `JobExecutionService`, and check that the counter got incremented on a correct member.